dsl_lang/lib/ast.ml

```diff
@@ -1,21 +1,14 @@
 type clss = string
 type var = string
 
-type declare =
-| DeclareClasses of clss list
-
 type policy =
-| Class of clss
-| Fifo of policy list
-| Fair of policy list
-| Strict of policy list
-| Var of var
-
-type return =
-| Return of policy
-
-type assignment =
-| Assn of var * policy
-
-type program =
-| Prog of declare * (assignment list) * return
\ No newline at end of file
+  | Class of clss
+  | Fifo of policy list
+  | Fair of policy list
```

Comment:
While we're here let's just rename `Fair` into `RoundRobin` to be more consistent with the glossary. The sample programs already use `rr`.

Reply:
I think #41 handles this (per our discussion in #31)